.github/workflows/build.yaml

```diff
@@ -59,8 +59,27 @@ jobs:
         uses: renovatebot/internal-tools@v1.3.6
         with:
           command: docker-builder
-          build-args: TARGET,BASE_IMAGE,USER_NAME
+          build-args: TARGET,USER_NAME
           last-only: true
           tag-suffix: ${{ env.TARGET }}
           major-minor: false
           dry-run: ${{ env.DRY_RUN }}
+
+
+  distro:
+    runs-on: ubuntu-latest
+
+    timeout-minutes: 15
+
+    strategy:
+      matrix:
+        distro:
+          [
+            bionic,
```

Comment:
Is this bionic-only because it's an additional test to verify the build?

Reply:
It's preparation for other distros later maybe for next ubuntu 22.04 or any other

Reply:
I'm thinking:
- Ensure a Dockerfile build with latest tools works on all supported editions
- Only do deep testing (e.g. lockfile generation) with the latest supported edition (e.g. 20.04 now, 22.04 later)

Reply:
This is now the case, test only installs latest tools for bionic, no lockfile generation. So this only verifies that all tools can be installed and are executable